packages/patternfly-4/react-core/src/components/Tooltip/Tooltip.d.ts

```diff
@@ -19,9 +19,9 @@ export interface TooltipProps extends Omit<HTMLProps<HTMLDivElement>, 'content'
   /** Tooltip additional class */
   className?: string;
   /** The reference element to which the tooltip is relatively placed to */
-  children: ReactElement<any>;
+  children: React.ReactNode;
   /** Tooltip content */
-  content: ReactElement<any> | string;
+  content: React.ReactNode;
```

Comment:
Curios about this one too.

Reply:
ChipGroup uses a Tooltip and this fixed the types for it.